lib/glsl-preview-view.js

```diff
@@ -69,9 +69,8 @@ const defaultUniforms = `
 `;
 
 module.exports = class GlslPreviewView {
-  constructor({ editorId, filePath }) {
+  constructor({ editorId }) {
```

Comment:
Yeah, I guess that's the default airbnb lint rule. I can add overrides, but I'm mostly just leaving that alone for now until something really bothers me. Eslint seems pretty good at just auto-fixing stuff like that.